parmed/amber/titratable_residues.py

```diff
@@ -159,37 +160,33 @@ def _str_refenes(self, solvent=False, igb=2, dielc=1.0):
         return ret_str
 
     def __str__(self):
-        if (self.typ=="ph"):
-            ret_str = ('%-4s\tpKa = %5.1f\n%8s' % (self.resname, self.pKa, 'ATOM') +
-                       ''.join(['%12s' % ('STATE %d' % i) for i in
-                               range(len(self.states))]) + '\n'
-            )
-        elif (self.typ=="redox"):
-            ret_str = ('%-4s\tEo = %7.3f V\n%8s' % (self.resname, self.pKa, 'ATOM') +
-                       ''.join(['%12s' % ('STATE %d' % i) for i in
-                               range(len(self.states))]) + '\n'
-            )
+        ret_str = ('%-4s\n%8s' % (self.resname, 'ATOM') +
+                   ''.join(['%12s' % ('STATE %d' % i) for i in
+                           range(len(self.states))]) + '\n'
+        )
         for i, atom in enumerate(self.atom_list):
             ret_str += ('%8s' % atom +
                         ''.join(['%12.4f' % (state.charges[i])
                                 for state in self.states]) + '\n'
             )
         ret_str += '-' * (8 + 12 * len(self.states)) + '\n'
-        if (self.typ=="ph"):
+        if (self.typ=="ph" or self.typ=="phredox"):
```

Comment:
Put spaces around the `==`. This needs to be done in lots of places.

Reply:
I fixed this.